src/pages/workspace/accounting/intacct/import/SageIntacctMappingsTypePage.tsx

```diff
@@ -23,33 +23,44 @@ function SageIntacctMappingsTypePage({route}: SageIntacctMappingsTypePageProps)
     const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${route.params.policyID ?? '-1'}`);
     const policyID = policy?.id ?? '-1';
     const mappings = policy?.connections?.intacct?.config?.mappings;
+    const exportConfig = policy?.connections?.intacct?.config?.export;
 
-    const selectionOptions = useMemo<SelectorType[]>(
-        () => [
-            {
+    const selectionOptions = useMemo<SelectorType[]>(() => {
+        const mappingOptions: SelectorType[] = [];
+        if (
+            mappingName !== CONST.SAGE_INTACCT_CONFIG.MAPPINGS.CUSTOMERS &&
+            mappingName !== CONST.SAGE_INTACCT_CONFIG.MAPPINGS.PROJECTS &&
```

Comment:
Nit: maybe we could build an array with those consts and check if it includes mapping 🤔 
But there are only two of them so it's up to you :)

```suggestion
            ![CONST.SAGE_INTACCT_CONFIG.MAPPINGS.CUSTOMERS, 
            CONST.SAGE_INTACCT_CONFIG.MAPPINGS.PROJECTS
            ].includes(mappingName) &&
```